src/main/resources/db/changelog/create/0018_EIP1-9508_add_status_gss_code_indexes.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="stephen.penney@softwire.com" id="0018_EIP1-9508_add_status_gss_code_index_to_postal"
+               context="ddl">
+
+        <createIndex tableName="postal_vote_application"
+                     indexName="postal_vote_application_status_gss_code_idx">
+            <column name="status"/>
```

Comment:
I think it would be wise to add date_created to this index too. At the moment, the engine is choosing between using an index on only GSS code and an index on only date created, to find, among applications with a certain GSS code and status, the N largest date_created values. Adding a GSS code / status index will improve the situation, but not guarantee that after a certain point again it decides that using the date index is a better plan.

My preference would be an index on (gss_code, status, date_created) in that order.